AudioBar/AudioBarViewController.swift

```diff
@@ -22,7 +22,10 @@ public final class AudioBarViewController: UIViewController, Elm.Delegate {
     @IBOutlet private var audioRouteView: UIView!
 
     @IBAction func userDidTapPlayPauseButton() {
-        program.dispatch(.togglePlay)
+        switch program.view.playPauseButtonMode {
```

Comment:
Did a different implementation since we don't won't to have any message in `program.view.playPauseButtonMessage`.